app/Services/Manager.php

```diff
@@ -608,4 +608,24 @@ public function sendEmail($user, $contest, $params)
             return null;
         }
     }
+
+    /**
+     * Finds if user exists in a competition for a given contest
+     *
+     * @param app\Models\Contest         $contest
+     * @param string                     $id
+     * @return array|null
+     */
+    public function findUserInContest($contest, $id)
+    {
+        $userCompetitions = $contest->competitions->filter(function ($competition, $key) use ($id) {
+            $users = $competition->users->where('northstar_id', $id);
```

Comment:
@weerd ya we "technically" only allow one person per competition, but we haven't had any actual logic enforcing that until now, so I thought it was better to not make the assumption that we would only get one thing back.